content/05.main-text.md

```diff
@@ -300,21 +354,49 @@ TODO: somewhere link to notebook https://github.com/greenelab/hetmech/blob/04206
 
 Columns stored in degree-perm summary files:
 source_degree target_degree n nnz sum sum_of_squares n_perms
+
+Cite data @doi:10.5281/zenodo.1435834
 -->
 
 ### Gamma-hurdle distribution
 
+<!-- references
+  https://nbviewer.jupyter.org/github/zietzm/hetmech/blob/afde1c6c2a7dea7db370ab543e0abcb992d34b0b/explore/gamma-hurdle-heatmaps/gamma-heatmaps.ipynb
+  https://github.com/greenelab/hetmech/pull/157
+  https://github.com/greenelab/hetmech/blob/088dc3c0852387b58880c145f265fd704eb5e5d1/explore/gamma-hurdle/parameter_estimates.ipynb
+  https://github.com/greenelab/hetmech/issues/123
+-->
+
+### Empirical DWPC p-values
```

Comment:
@ben-heil this section here summarizes the empirical DWPC p-values implementation you helped with if you'd like to read it over.

Reply:
Looks good to me